packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/UpgradeRolesForm/UpgradeRolesForm.tsx

```diff
@@ -84,26 +85,26 @@ export const UpgradeRolesForm = ({
     try {
       const response = await axios.post(`${app.serverUrl()}/upgradeMember`, {
         new_role: newRole,
-        // @ts-expect-error <StrictNullChecks/>
-        address: _user.Address.address,
+        // TODO: allow setting any address (??)
+        address: _user.addresses[0].address,
```

Comment:
same as [this](https://github.com/hicommonwealth/commonwealth/pull/8509/files#r1691535063)